fitlins/interfaces/bids.py

```diff
@@ -103,8 +102,9 @@ def _run_interface(self, runtime):
         from bids.analysis import auto_model
         models = self.inputs.model
         if not isinstance(models, list):
+            database_path = self.inputs.database_path
             # model is not yet standardized, so validate=False
-            layout = bids.BIDSLayout(self.inputs.bids_dir, validate=False)
+            layout = bids.BIDSLayout.load(database_path=database_path)
```

Comment:
The use of `.load` here makes sense, as we have already "validated" that the CLI options match the db path.